dnf.spec

```diff
@@ -289,13 +289,13 @@ popd
 %dir %{confdir}/modules.d
 %dir %{confdir}/modules.defaults.d
 %dir %{pluginconfpath}
-%if 0%{?fedora} <= 38
+%if 0%{?fedora} <= 38 || 0%{?rhel} <= 9
 %dir %{confdir}/protected.d
 %dir %{confdir}/vars
 %endif
 %dir %{confdir}/aliases.d
 %exclude %{confdir}/aliases.d/zypper.conf
-%if 0%{?fedora} <= 38
+%if 0%{?fedora} <= 38 || 0%{?rhel} <= 9
```

Comment:
```suggestion
%if ! (0%{?fedora} > 38 || 0%{?rhel} > 9)
```

Reply:
Thanks. Addressed in https://github.com/rpm-software-management/dnf/pull/1975.